t/shdict.t

```diff
@@ -999,3 +999,26 @@ number expected, got string
 --- no_error_log
 [alert]
 [crit]
+
+
+
+=== TEST 31: get stats
+--- http_config eval: $::HttpConfig
+--- config
+    location = /t {
+        content_by_lua_block {
+            local dogs = ngx.shared.dogs
+            local total_used, total_size = dogs:stats()
+            ngx.say("total_used type: ", type(total_used))
+            ngx.say("total_size type: ", type(total_size))
```

Comment:
Can we do more strict checks here? For one, the numbers returned should all be non-negative integer values. Also, the first value must be smaller than the second. Finally, the total size should be predetermined and is a constant (though may differ on 32-bit and 64-bit systems, need to be prepared for that).

Also I think the first value should be a constant for this particular test case and it only differs on 32-bit and 64-bit systems and we just accept those 2 different constants in the test.